cosmo_tester/resources/infrastructure_blueprints/aws/vm-ipv6-net.yaml

```diff
@@ -0,0 +1,148 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+imports:
+  - https://cloudify.co/spec/cloudify/5.1.0/types.yaml
+  - plugin:cloudify-aws-plugin
+
+inputs:
+  test_infrastructure_name: {}
+  security_group_id: {}
+  subnet_id: {}
+  vpc_id: {}
+  name_filter:
+    default:
+      Name: name
+      Values:
+        - "*"
+  image_owner:
+    default: "*"
+  image_id:
+    default: "*"
+  flavor: {}
+  userdata:
+    default: ""
+  use_net:
+    default: "1"
+
+dsl_definitions:
+  client_config: &client_config
+    aws_access_key_id: { get_secret: aws_access_key_id }
+    aws_secret_access_key: { get_secret: aws_secret_access_key }
+    region_name: { get_secret: ec2_region_name }
+
+node_templates:
+  test_host:
+    type: cloudify.nodes.aws.ec2.Instances
+    properties:
+      agent_config:
+        install_method: none
+      use_ipv6_ip: True
+      resource_config:
+        ImageId: { get_attribute: [ image, aws_resource_id ] }
+        InstanceType: { get_input: flavor }
+        kwargs:
+          KeyName: { get_input: test_infrastructure_name }
+          UserData: { get_input: userdata }
+      client_config: *client_config
+      Tags:
+        - Key: Name
+          Value: { get_input: test_infrastructure_name }
+        - Key: Owner
+          Value: { get_secret: owner_tag }
+    relationships:
+      - type: cloudify.relationships.depends_on
+        target: nic
+      - type: cloudify.relationships.depends_on
+        target: ip
+      - type: cloudify.relationships.depends_on
+        target: image
+
+  nic:
+    type: cloudify.nodes.aws.ec2.Interface
+    properties:
+      client_config: *client_config
+      resource_config:
+        kwargs:
+          Groups:
+            - { get_attribute: [ security_group, aws_resource_id ] }
+      Tags:
+        - Key: Name
+          Value: { concat: [ { get_input: test_infrastructure_name }, "-", { get_input: use_net } ] }
+        - Key: Owner
+          Value: { get_secret: owner_tag }
+    relationships:
+      - type: cloudify.relationships.depends_on
+        target: subnet
+      - type: cloudify.relationships.depends_on
+        target: security_group
+
+  ip:
+    type: cloudify.nodes.aws.ec2.ElasticIP
+    properties:
+      client_config: *client_config
+      Tags:
+        - Key: Name
+          Value: { concat: [ { get_input: test_infrastructure_name }, "-", { get_input: use_net } ] }
+        - Key: Owner
+          Value: { get_secret: owner_tag }
+    relationships:
+      - type: cloudify.relationships.depends_on
+        target: nic
+
+  subnet:
+    type: cloudify.nodes.aws.ec2.Subnet
+    properties:
+      use_external_resource: true
+      resource_id: { get_input: subnet_id }
+      client_config: *client_config
+      resource_config:
+        CidrBlock: 192.168.42.0/24
```

Comment:
Why do you have an IPv4 block in your IPv6 blueprint? I'm assuming it's because of the floating IPs, but maybe add a comment to that effect if that's right?

Reply:
Yup, it seems some form of IPv4 is necessary for the floating/elastic IP to work.  Comment added

Reply:
That's a pity but let's not worry about it more right now.